src/plugins/rest/rest.component.ts

```diff
@@ -0,0 +1,37 @@
+import {Component, Optional, Input, OnInit, OnDestroy} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {Rest as RestView} from 'ng2-qgrid/plugin/rest/rest';
+import {RootService} from '../../infrastructure/component/root.service';
+import {PluginComponent} from '../plugin.component';
+
+@Component({
+	selector: 'q-grid-rest',
+	template: ''
+})
+export class RestComponent extends PluginComponent implements OnInit, OnDestroy {
+	@Input('url') public restUrl: string;
+	@Input('method') public restMethod: string;
+	@Input('serialize') public restSerialize: (x: any) => any;
+
+	private rest: RestView;
+
+	constructor(private http: HttpClient, @Optional() root: RootService) {
+		super(root);
+
+		this.models = ['rest'];
+	}
+
+	ngOnInit() {
+		super.ngOnInit();
+
+		const {method, url, serialize} = this.model.rest();
+		this.rest = new RestView(this.model, {
```

Comment:
no need to pass url, method, serialize, it should be inside the model automatically

Reply:
remove `url, method, serialize` from here